File: 4. Java/EX09.java
package Apostila;

import java.util.Scanner;
import static java.lang.Math.pow;
import java.util.Locale;

public class EX09
{
    public static void main(String[] args)
    {
        // Declara variáveis
        double areQua, areTri, areTra;
        double a, b, c;        
        
        // Define localização
        Locale.setDefault(Locale.US);
        Scanner ent = new Scanner(System.in);
        
        // Entrada de dados
        System.out.print("# Digite a medida A: ");
        a = ent.nextDouble();
        
        System.out.print("# Digite a medida B: ");
        b = ent.nextDouble();
        
        System.out.print("# Digite a medida C: ");
        c = ent.nextDouble();
        
        // Calcula áreas
        areQua = pow(a, 2);              // Área do quadrado
        areTri = (a * b) / 2;            // Área do triângulo
        areTra = ((a + b) * c) / 2;      // Área do trapézio

        // Exibe os resultados formatados
        System.out.println("\n# AREA DO QUADRADO = " + String.format("%.4f", areQua));
        System.out.println("# AREA DO TRIANGULO = " + String.format("%.4f", areTri));
        System.out.println("# AREA DO TRAPEZIO = " + String.format("%.4f", areTra));
    }
}

File: 6. Python/EX09.py
from math import pow

# Função para exibir o relatório das áreas
def relatorio(areQua, areTri, areTra):
    print(f"* AREA DO QUADRADO = {areQua:.4f}")     # Exibe a área do quadrado
    print(f"* AREA DO TRIANGULO = {areTri:.4f}")    # Exibe a área do triângulo
    print(f"* AREA DO TRAPEZIO = {areTra:.4f}")     # Exibe a área do trapézio

def main():
    # Solicita as medidas
    a = float(input("# Digite a medida A: "))
    b = float(input("# Digite a medida B: "))
    c = float(input("# Digite a medida C: "))

    # Calcula as áreas
    areQua = pow(a, 2)          # Área do quadrado
    areTri = (a * b) / 2        # Área do triângulo
    areTra = ((a + b) * c) / 2  # Área do trapézio

    # Chama a função para exibir o relatório
    relatorio(areQua, areTri, areTra)

if __name__ == '__main__':
    main()  # Executa a função principal
